docs/languages/julia.md

```diff
@@ -0,0 +1,71 @@
+# Julia in Visual Studio Code
+
+The [Julia extension](https://marketplace.visualstudio.com/items?itemName=julialang.language-julia) for Visual Studio Code includes built-in dynamic autocompletion, inline results, plot pane, integrated REPL, variable view, code navigation, and many other advanced language features.
```

Comment:
I'm not familiar with the Julia language but I think it is popular for data science.
It might help search discovery and set context by mentioning it's uses in data science or other domains?

Reply:
Added a section for this